pg_shard.c

```diff
@@ -378,6 +378,25 @@ ErrorIfQueryNotSupported(Query *queryTree)
 						errmsg("unsupported utility statement")));
 	}
 
+	/*
+	 * Reject subqueries which are in SELECT or WHERE clause.
+	 * Queries which include subqueries in FROM clauses are rejected below.
+	 */
+	if (queryTree->hasSubLinks == true)
+	{
+		ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+						errmsg("cannot perform distributed planning on this query"),
```

Comment:
I know you're just copy-pasting this message, but can we change it to: _cannot perform distributed planning for the given query_? The use of _this query_ in an error message isn't common within PostgreSQL.